pom.xml

```diff
@@ -2869,6 +2869,7 @@
               <derby.system.durability>test</derby.system.durability>
               <java.awt.headless>true</java.awt.headless>
               <java.io.tmpdir>${project.build.directory}/tmp</java.io.tmpdir>
+              <spark.hadoop.hadoop.caller.context.enabled>true</spark.hadoop.hadoop.caller.context.enabled>
```

Comment:
Got it. So, previously, I added this to `maven-surefire-plugin` only and missed `scalatest-maven-plugin`. Thank you, @cnauroth !